Octokit/Models/Common/PushWebhookCommitter.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
```

Comment:
I would say this should live under Models/Response/ActivityPayloads with the others